ui/DesignSystem/Component/Sidebar.svelte

```diff
@@ -15,16 +15,16 @@
 </script>
 
 <style>
```

Comment:
Not a huge fan of having 2 `Sidebar.svelte`'s in the project; maybe the layout one could be named something else? Like `MainLayout` or `HomeLayout` or `DefaultLayout`?